components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -377,6 +377,9 @@ export const WORKSPACE_TIMEOUT_EXTENDED: WorkspaceTimeoutDuration = "180m";
 export const WORKSPACE_LIFETIME_SHORT: WorkspaceTimeoutDuration = "8h";
 export const WORKSPACE_LIFETIME_LONG: WorkspaceTimeoutDuration = "36h";
 
+export const MAX_PARALLEL_WORKSPACES_FREE = 4;
```

Comment:
Was skeptical why to pull them here, but saw it's used for explaining the behavior in UI :+1: